ui/src/helpers/api.js

```diff
@@ -0,0 +1,51 @@
+import * as axios from "axios";
+
+const mizuAPIPathPrefix = "/mizu";
+
+// When working locally (with npm run start) change to:
+// export const MizuWebsocketURL = `ws://localhost:8899${mizuAPIPathPrefix}/ws`;
+export const MizuWebsocketURL = `ws://${window.location.host}${mizuAPIPathPrefix}/ws`;
+
+export default class Api {
+    constructor() {
+        this.client = null;
+        // When working locally (with npm run start) change to:
+        // this.api_url = `http://localhost:8899/${mizuAPIPathPrefix}/api/`;
+        this.api_url = `${window.location.origin}${mizuAPIPathPrefix}/api/`;
+    }
+
+    init = () => {
```

Comment:
why not init this in the constructor and then just use this.client (that will be already initialize)?